domains/models.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import unicode_literals
+import uuid
+
+from django.contrib.auth.models import User
+from django.db import models
+
+
+AUTHORIZATION_STATUS_CHOICES = (
+    ('unknown', 'unknown'),
+    ('pending', 'pending'),
+    ('processing', 'processing'),
+    ('valid', 'valid'),
+    ('invalid', 'invalid'),
+    ('revoked', 'revoked')
+)
+
+AUTHORIZATION_TYPE_CHOICES = (
```

Comment:
Taken from the [ACME protocol Identifier Validation Challenges](https://letsencrypt.github.io/acme-spec/#identifier-validation-challenges) types.